src/explorer/editors/FileEditor.ts

```diff
@@ -26,8 +25,9 @@ export class FileEditor extends BaseEditor<IAzureNode<FileTreeItem>> {
 
     async getData(node: IAzureNode<FileTreeItem>): Promise<string> {
         const webAppClient = nodeUtils.getWebSiteClient(node);
-        const user = await util.getWebAppPublishCredential(webAppClient, node.treeItem.site);
-        const kuduClient = new KuduClient(node.treeItem.site.name, user.publishingUserName, user.publishingPassword);
+        const publishingCredential = await node.treeItem.siteWrapper.getWebAppPublishCredential(webAppClient);
+        const kuduClient = new KuduClient(node.treeItem.siteWrapper.name, publishingCredential.publishingUserName, publishingCredential.publishingPassword);

```

Comment:
Shouldn't this be `siteWrapper.appName`? Othewise I don't think it'll work for slots